pkg/roll/execute.go

```diff
@@ -179,7 +207,9 @@ func (m *Roll) createView(ctx context.Context, version, name string, table schem
 	}
 
 	_, err := m.pgConn.ExecContext(ctx,
-		fmt.Sprintf("CREATE OR REPLACE VIEW %s.%s %s AS SELECT %s FROM %s",
+		fmt.Sprintf("BEGIN; DROP VIEW IF EXISTS %s.%s; CREATE VIEW %s.%s %s AS SELECT %s FROM %s; COMMIT",
```

Comment:
why the change from `create or replace` to `drop and create`?

Reply:
I found out that changes adding new columns to an existing view (because of a SQL op), could fail to `create or replace` the view, as it requires all columns to be in the same order. Since this is equivalent and doesn't fail I went for it